boards/arm/ubx_evkannab4_nrf52833/ubx_evkannab4_nrf52833_defconfig

```diff
@@ -0,0 +1,30 @@
+# ANNA-B4 EVK board configuration
+#
+# Copyright (c) 2021 u-blox AG
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52833_QIAA=y
+CONFIG_BOARD_UBX_EVKANNAB4_NRF52833=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable hardware stack protection
+CONFIG_HW_STACK_PROTECTION=y
+
+# Enable RTT
+CONFIG_USE_SEGGER_RTT=y
```

Comment:
this is a non-minimal setting, so should likely be removed (applications can enable it as needed)

Reply:
This part is similar in many of the boards (nrf52833dk_nrf52833, nrf52840dk_nrf52840 etc), should I change here or should there be a common change for all of them?